vagrant/base/elk/Vagrantfile

```diff
@@ -0,0 +1,43 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box     = "precise64"
+  config.vm.box_url = "http://files.vagrantup.com/precise64.box"
+#  config.vm.box = "geerlingguy/ubuntu1404"
+  config.ssh.insert_key = false
+
+  config.vm.provider :virtualbox do |v|
+    v.memory = 1024
+    v.cpus = 2
+    v.customize ["modifyvm", :id, "--natdnshostresolver1", "on"]
+    v.customize ["modifyvm", :id, "--ioapic", "on"]
+  end
+
+# ELK server.
+  config.vm.define "logs" do |logs|
+    logs.vm.hostname = "logs"
+    logs.vm.network :private_network, ip: "192.168.9.90"
+
+    logs.vm.provision :ansible do |ansible|
+      ansible.playbook = "../../../playbooks/vagrant-elk.yml"
+      ansible.inventory_path = "../../../../ansible-vagrant-examples/elk/provisioning/elk/inventory"
```

Comment:
Actually, it looks like there's already a `playbooks/vagrant/inventory.ini` file; what about moving the contents there and then pointing to it?